.github/workflows/ci.yml

```diff
@@ -139,7 +139,8 @@ jobs:
             ./docker-bake.json
             ./docker-bake.hcl
             ${{ steps.meta.outputs.bake-file }}
-          targets: use
+          # Include "latest" which is latest "use" image
+          targets: use latest
```

Comment:
Could we not restore the ability to tag `latest` with the name of the branch too? This was useful for manually triggered jobs. Arguably the fact these manually triggered jobs clobber the `use-*` tags but not `latest` could be confusing.

Reply:
We could. How was it useful?

Regardless I'll treat it as a separate issue/PR to get this in sooner and unblock https://github.com/Agoric/agoric-sdk/pull/8873

Reply:
It was useful to generate special "forked" images, but I only used it for generating use layer images.